### ISSUE TITLE
[BUG] Container releases not following GitHub releases

### ISSUE THREAD

**Issue by codekow** [bugfest/tor-controller#53]
**Describe the bug**
New container image tags are not available in quay for releases after 0.9.0

**To Reproduce**
See:
- https://quay.io/repository/bugfest/tor-daemon?tab=tags
- https://quay.io/repository/bugfest/tor-controller?tab=tags

**Expected behavior**
Expect to see container image tags updated for each release

**Comment by bugfest** [comment]
Hi @codekow, the latest container images are tagged as `0.9.0`, tor-controller releases are in reality the helm chart version; latest version is `0.1.11`. Full version table is documented at https://github.com/bugfest/tor-controller#versions

Please let me know if I this answer your case.

**Comment by codekow** [comment]
Maybe I'm seeing another issue or I don't understand the release cycle.

I would assume each container image is versioned with the release version in the repo. 

There are 4 Dockerfile(s) in this repo:
- [Dockerfile](https://github.com/bugfest/tor-controller/blob/master/Dockerfile)
- [Dockerfile.tor-daemon](https://github.com/bugfest/tor-controller/blob/master/Dockerfile.tor-daemon)
- [Dockerfile.tor-daemon-manager](https://github.com/bugfest/tor-controller/blob/master/Dockerfile.tor-daemon-manager)
- [Dockerfile.tor-onionbalance-manager](https://github.com/bugfest/tor-controller/blob/master/Dockerfile.tor-onionbalance-manager)

I would expect the tags in quay to match the release since the release may have changed the container image. I'm not seeing those tags.

**Comment by codekow** [comment]
I would avoid tagging container images based on the helm version. That is very confusing.

**Comment by bugfest** [comment]
Hi @codekow, 

We're not using image tags to tag the helm version; they're following different version numbers. All the containers are tagged based on the functionallities we're releasing; probably you're confussed due to the fact that we're using 2 tag formats in parallel. 

This project builds 2 artifact types: containers and a helm chart. The later is hosted in gh pages. To achieve it we use two tag formats (https://github.com/bugfest/tor-controller/tags): 
- `x.x.x` (to tag containers) - https://github.com/bugfest/tor-controller/blob/5e88648b35e4a575afdec68d120978bf4b12904a/.github/workflows/main-tag.yml#L6
- `tor-controller-y.y.y` (to tag helm chart) - automatically created by https://github.com/bugfest/tor-controller/blob/master/.github/workflows/release.yml

Hope this clarifies it.
